presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1284,6 +1295,100 @@ public void revokeTablePrivileges(MetastoreContext metastoreContext, String data
         }
     }
 
+    @Override
+    public long lock(MetastoreContext metastoreContext, String databaseName, String tableName)
+    {
+        try {
+            final LockComponent lockComponent = new LockComponent(LockType.EXCLUSIVE, LockLevel.TABLE, databaseName);
```

Comment:
static import `LockType.EXCLUSIVE`, `LockLevel.TABLE`

Reply:
`TABLE` has been imported by `org.apache.hadoop.hive.metastore.api.HiveObjectType.TABLE`, so I left `LockLevel.TABLE` as is.